src/packages/core/src/external/githubCorner.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+export interface GithubCornerProps {
+  githubUrl: string;
+  fill?: string;
+  color?: string;
+  width?: number;
+  height?: number;
+}
+
+
+export const GithubCorner = (props: GithubCornerProps): React.ReactElement => {
+  const StyledLink = styled.a`
+  cursor: pointer;
+  z-index: 9999;
+  `;
+
+  const StyledSvg = styled.svg`
+  fill: ${props.fill};
```

Comment:
these props should be passed into the component, see https://github.com/kibalabs/ui-react/blob/main/src/atoms/link/component.tsx for an example